saleor/graphql/order/mutations/fulfillment_update_tracking.py

```diff
@@ -36,6 +38,12 @@ class Meta:
         permissions = (OrderPermissions.MANAGE_ORDERS,)
         error_type_class = OrderError
         error_type_field = "order_errors"
+        webhook_events_info = [
+            WebhookEventInfo(
+                type=WebhookEventAsyncType.FULFILLMENT_UPDATED,
```

Comment:
I am not sure if the scope of the event here is correct. It is a single place where you added it. The mutation is not standard "update" action. It is an update of the tracking number.
If you don't plan to add more fulfillment_updated events, we should have an event for updating the tracking number instead of fulfillment_updated. (Also `tracking_number_updated` [I know, the name is not the best one]  is already implemented on PluginsManager side - see `fulfillment_tracking_updated`)

Reply:
cc @maarcingebala

Reply:
ok so you suggest that we should have 2 separate events, one for tracking number update, and the second for general fulfillment update, right? and add the possibility for user to choose which one of those two he would like to receive? also if the new `FULFILLMENT_UPDATE` event is used only in this place - for mutation confirming a tracking number - better is to rename it for `fulfillmeent_tracking_number_updated`

Reply:
After discussing the topic internally, we decided to introduce a new webhook event dedicated to setting tracking number: https://github.com/saleor/saleor/issues/13670.